week1/exercises-week1/src/ex2/Guarantee.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import GuaranteeComp from "./GuaranteeComp";
+
+// For some reason it doesn't render images.
+
+function Guarantee() {
+  return (
+    <div style={{ display: "flex" }}>
+      <GuaranteeComp
+        content={{
```

Comment:
No need to create a content property here. You can pass multiple properties to the component
```suggestion
      <GuaranteeComp
         imgUrl={...}
         title={...}
         description={...}
```